crates/vm/levm/src/call_frame.rs

```diff
@@ -80,15 +81,23 @@ pub struct CallFrame {
     pub transient_storage: TransientStorage,
     pub logs: Vec<Log>,
     pub depth: usize,
+    pub valid_jump_destinations: HashSet<usize>,
 }
 
 impl CallFrame {
     pub fn new_from_bytecode(bytecode: Bytes) -> Self {
-        Self {
+        Self::new(
+            Default::default(),
+            Default::default(),
+            Default::default(),
             bytecode,
-            gas_limit: U256::MAX,
-            ..Default::default()
-        }
+            Default::default(),
+            Default::default(),
+            Default::default(),
+            U256::MAX,
+            Default::default(),
+            Default::default(),
+        )
```

Comment:
You don't need to write all those `Default::default()`s, you can keep the previous implementation and just add the bytecode field.

Reply:
Done in [Return to new_from_bytecode previous version](https://github.com/lambdaclass/ethrex/pull/1434/commits/00f194205f4db0d4dfa8e155e355f68be71ecfa1)